test/verify/check-metrics

```diff
@@ -247,9 +247,14 @@ class TestHistoryMetrics(testlib.MachineCase):
         b.wait_in_text("#metrics-hour-1597662000000.metrics-hour-compressed .spikes_count", "3 spikes")
         b.wait_in_text("#metrics-hour-1597662000000.metrics-hour-compressed .spikes_info", "1 Memory, 1 Disk I/O, 1 Network I/O")
 
+        # HACK: the bottom panel has some unpredictable extra white line at the bottom, but is also uninteresting
+        b.eval_js("document.querySelector('.bottom-panel').style.display = 'none'")
```

Comment:
Can we use `mock` for this?
```
mock={ ".bottom-panel": "" }
```

Reply:
That's a great idea, but unfortunately not. It produces this:

![TestHistoryMetrics-testEvents-metrics-history-compressed-hour-medium-pixels](https://github.com/user-attachments/assets/a5f6f05a-fdc8-4424-a5ce-98b2cf303863)

which valdalizes the button, but keeps the problematic space.